### PR TITLE
feat: run npm install after init

### PR DESCRIPTION
**Pull request by JustinBeckwith**
Addresses #90

### CODE REVIEW

File: src/init.ts [hunk 1]
```diff
@@ -215,5 +215,9 @@ export async function init(options: Options): Promise<boolean> {
     options.logger.log('No edits needed in package.json.');
   }
   await generateTsConfig(options);
+
+  // Run `npm install` after initial setup so `npm run check` works right away
+  cp.spawnSync('npm', ['install'], {stdio: 'inherit'});
```
Comment (ofrobots) [inline src/init.ts]: Question:
* Should this happen if we didn't actually modify dependencies? My guess is yes.
* Should this happen if the user passed `--dry-run`? My guess is no.
* Can you add a test to `test-kitchen`?

Comment (JustinBeckwith) [reply src/init.ts]: Answers!
- Yes, I think so.
- Nope, fixed :)
- Done.  Was more of an adventure than I expected.

**Comment by JustinBeckwith** [comment]
@ofrobots testing this turned into an adventure.  Turns out we weren't really testing `npx init` at all.  To make that work, I needed to add a `-n` flag.  PTAL!

File: test/test-kitchen.ts [hunk 2]
```diff
@@ -71,11 +71,18 @@ test.before(async () => {
...
   await ncpp('test/fixtures', `${stagingPath}/`);
-  await simpleExecp('npm install', execOpts);
 });
 
 test.serial('init', async t => {
-  await simpleExecp('./node_modules/.bin/gts init -y', execOpts);
+  const nodeVersion = Number(process.version.slice(1).split('.')[0]);
+  if (nodeVersion < 8) {
+    await simpleExecp('npm install', execOpts);
+    await simpleExecp('./node_modules/.bin/gts init -y', execOpts);
```
Comment (ofrobots) [inline test/test-kitchen.ts]: Q: Any reason why there is `-y` on this leg of the branch and a `-n` on the other? If there is a valid reason, this deserves a comment.

Comment (JustinBeckwith) [reply test/test-kitchen.ts]: Added a comment.  You could use -y or -n in the < 8 version, but you must use -n in the > 8 version.

**Pull request merged**